### PR TITLE
add get instance state handler

### PR DESCRIPTION
**Pull request by nlu90**

### CODE REVIEW

File: heron/ckptmgr/src/cpp/manager/ckptmgr-server.cpp [hunk 1]
```diff
@@ -113,6 +114,43 @@ void CkptMgrServer::HandleSaveInstanceStateRequest(REQID _id, Connection* _conn,
...
+
+    LOG(INFO) << "Get checkpoint success for " << checkpoint.getCkptId() << " "
+              << checkpoint.getComponent() << " " << checkpoint.getInstance();
+  } else {
+    LOG(INFO) << "Get checkpoint failed for " << checkpoint.getCkptId() << " "
+              << checkpoint.getComponent() << " " << checkpoint.getInstance();
+  }
+
+  SendResponse(_id, _conn, response);
+  delete _req;
```
Comment (maosongfu) [inline heron/ckptmgr/src/cpp/manager/ckptmgr-server.cpp]: @srkukarni Are we to release within the mempool or to delete directly?

Comment (srkukarni) [inline heron/ckptmgr/src/cpp/manager/ckptmgr-server.cpp]: preferable to use __global_protobuf_pool_release__

Comment (nlu90) [inline heron/ckptmgr/src/cpp/manager/ckptmgr-server.cpp]: OK. 

Just a reminder that I found stmgr-server.cpp is mix-using both. Do we also need to clean those `delete` used there?

Comment (srkukarni) [inline heron/ckptmgr/src/cpp/manager/ckptmgr-server.cpp]: at some point yes

**Comment by maosongfu** [comment]
This commit fails the building.

**Comment by srkukarni** [comment]
Fixed it

**Pull request merged**